driver.go

```diff
@@ -98,6 +98,27 @@ func (hd *hetznerDriver) Create(req *volume.CreateRequest) error {
 		}
 	}
 
+	uid := getOption("uid", req.Options)
+	gid := getOption("gid", req.Options)
+	if uid != "0" || gid != "" {
+		// string to int
+		uintParsed, err := strconv.Atoi(uid)
+		if err != nil {
+			logrus.Warnf("failed to parse uid option value as uint32: %s", uid)
+			uintParsed = 0
+		}
+		gidParsed, err := strconv.Atoi(gid)
+		if err != nil {
+			logrus.Warnf("failed to parse gid option value as uint32: %s", gid)
```

Comment:
nit: `strconv.Atoi` returns `int`, not `uint32` (also: the exact go type is probably irrelevant for the user seeing the error; I think the important part is the "integer")